sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/events/EventParticipantsPage.java

```diff
@@ -21,5 +21,5 @@
 import org.openqa.selenium.By;
 
 public class EventParticipantsPage {
-  public static final By ADD_PARTICIPANT_BUTTON = By.cssSelector("div#eventParticipantAddPerson");
```

Comment:
why did you delete this ?

Reply:
removed unused variables

Reply:
thats strange hopefully will not be deleted when merged into the dev branch :)

Reply:
on my branch is unused, but if is used in current development, when I'll do the rebase we'll have it back again :D

Reply:
@richardbartha just rebased, and added back the locator